Cesium3DTiles/src/Tileset.cpp

```diff
@@ -799,7 +799,7 @@ static BoundingVolume createDefaultLooseEarthBoundingVolume(
   context.requestHeaders.push_back(std::make_pair(
       "Accept",
       "application/vnd.quantized-mesh,application/octet-stream;q=0.9,*/"
-      "*;q=0.01"));
+      "*;q=0.01;extensions=watermask"));
```

Comment:
Should we make the watermask optional? It's a good amount of extra data to to download and won't be used at all in many applications.